renderer/types/actions/editor-actions.ts

```diff
@@ -0,0 +1,82 @@
+import * as consts from '../../consts';
+
+export interface editorUpdateAction {
+  type: consts.EditorActionsTypes.UPDATE_EDITOR;
+  code: string;
+};
+
+export interface openFileSucceededAction {
+  type: consts.EditorActionsTypes.OPEN_FILE_SUCCEEDED;
+  code: string;
+  filepath: string;
+}
```

Comment:
@HaileyJang Looks good, just a minor style thing. Can you capitalize these interface names? It's to indicate that these are types.